misc/discovery_schema.json

```diff
@@ -393,7 +393,7 @@
                             },
                             "num_oid": {
                                 "type": "string",
-                                "pattern": "^(\\.\\d+)+\\.?\\{\\{ \\$index \\}\\}(\\.\\d+)*$"
+                                "pattern": "^(\\.\\d+)+\\.?(\\{\\{ \\$index \\}\\})*(\\.\\d+)*$"
```

Comment:
So, I checked and most or all? of these are returning an oid without an index.

This means instead of modifying the detection, you can just add {{ $index }} at the end of your oid and it will insert an empty string. Like so `num_oid: '.1.3.6.1.4.1.22408.1.1.2.1.5.108.111.97.100.50.1{{ $index }}'`

I know it feels a little weird, but this detection is here for good reason.

Reply:
All of these are returning an OID without an index. Therefore I suspect OID should remain as a name, and num_oid should be either the same name OR numeric description of the same OID with {{ $index}} tagged on the end.

Reply:
setting num_oid to numeric index followed by {{.$index }} works and avoids the need to change test cases